docs/Calcom.md

```diff
@@ -0,0 +1,109 @@
+# Accept Bitcoin payment for your Cal.com bookings
+
+You can now accept Bitcoin payment for all your Calcom bookings and appointments. 
+Whether you're offering consultation services or one-on-one meetings, receive Bitcoin payments today directly in your wallet—no middlemen, no platform fees, and no hidden costs.
+
+
+
+## Prerequisites:
+
+Before diving into the setup process, ensure you have the following:
+
+- [Cal.com account](https://cal.com/)
+- BTCPay Server - [self-hosted](Deployment.md) or run by a [third-party host](/Deployment/ThirdPartyHosting.md) * v2.0.6 or later.
+- [Created BTCPay Server store](CreateStore.md) with [wallet set up](WalletSetup.md)
+
+
+## Set up Cal.com with BTCPay Server
+
+Login to your cal.com account. Navigate to `Apps` > `App store` > `Payments apps`
+
+Find the `BTCPayServer` app, click `details`, and then `install app` button
+
+![Cal.com: image 1](./img/calcom/1_app_install_details.png)
+
+Select the application you want to connect your BTCPay Server instance to
+
+![Cal.com: image 2](./img/calcom/2_installation_step_one.png)
+
+
+The next step is to fill in your BTCPay credentials. Open your BTCPay Server instance on a new tab
+
+**BTCPay Server URL**: your BTCPay instance url e.g. https://example.btcpay.com
+
+**BTCPay Store Id**: The store you want connected to cal.com. On your BTCPay server instance, select the chosen store, click on `Settings` on the left navigation, then you’d see your storeId displayed.
+
+Copy the storeId, and fill in your cal.com - BTCPay server installation form
```

Comment:
Pay attention to naming consistency. For example, Store Id should be refered to how it appears in the UI for the user.
```suggestion
Copy the Store Id, and fill in your Cal.com - BTCPay Server installation form
```